changes/features/pr.64.md

```diff
@@ -0,0 +1 @@
+Added changes to remove features duplicate for `add_features` and `mutation` parameter to mutate input features or not.
```

Comment:
Sounds a bit vague. ;) And maybe start with "Remove duplicate features for..."? Not sure.